### DISCUSSION TITLE
static export doesn't work?

### DISCUSSION THREAD

**Discussion by rmarsono** [jaredpalmer/razzle#1728] [Q&A]
Hello everyone!

I just started using Razzle. My use case is I need to be able to export my react app as HTML with embedded JS. Out of the box, Razzle seems to be able to do this and my app is running fine on localhost. When I export, the build folder is properly generated along with the js file but when I load this on the browser, the JS doesn't seem to work? 

I tried for example using codepen to copy paste HTML and JS and this doesn't work. Perhaps there is some dependency that I am missing? 

For context, I have my `static_export.js` set up as well as the `server.tsx` file which was taken straight from the examples but the exported files don't seem to work.

Please help.

**Comment by fivethreeo** [comment]
When you say load on the browser. You mean open the html  file in the  browser?
